frontend/src/pages/AdminCreationPage.css

```diff
@@ -0,0 +1,3 @@
+.layout-dropdowns {
+
```

Comment:
Wenn deine css-Klasse leer ist, lass sie weg. Sie kann einen Eslint-Fehler verursachen, wenn sie in deinem Projekt enthalten ist.